### ISSUE TITLE
featuremap size reduce in op.py FactorizedReduce function

### ISSUE THREAD

**Issue by yjump** [khanrc/pt.darts#18]
Hi all,
     Thank you very much for this repo. Here I have a question. In op.py line 178 FactorizedReduce function, why conv2 layer need to reduce size on dim=2 and 3? In my case this cause dimension mismatch. 
out = torch.cat([self.conv1(x), self.conv2(x[:, :, 1:, 1:])], dim=1)
